ampcrowd/docker-entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Migrate databases if necessary
+sleep 5
+python ampcrowd/manage.py syncdb --noinput
+
+# Process options
+
+# Enable SSL
+if [ "$1" == "-s" ] || [ "$2" == "-s" ]
+then
+    export SSL=1
+else
+    export SSL=0
+fi
+
+python ampcrowd/manage.py collectstatic --noinput
+
+# Print errors directly to console for easy debugging with -d
+if [ "$1" == "-d" ] || [ "$2" == "-d" ]
+then
+    export DEVELOP=1
+    python ampcrowd/manage.py celeryd -l DEBUG &
+    python ampcrowd/manage.py runserver 0.0.0.0:8000
```

Comment:
I like the auto-reload capabilities of the Django development server as well. Unfortunately, it doesn't work with SSL, so this change would make it impossible to run with both `-d` and `-s`, which is a common configuration when trying to debug the AMT integration.